nativecfg/resource-config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "resources":{
+  "includes":[
+    {"pattern":"\\Qclojure/core.clj\\E"}, 
```

Comment:
Resource config should be trimmed to:
https://github.com/FieryCod/holy-lambda/blob/master/modules/holy-lambda-babashka-tasks/src/holy_lambda/tasks.clj#L749